openpgp.d.ts

```diff
@@ -22,6 +22,8 @@ export function generateSessionKey(options: { encryptionKeys: PublicKey[], date?
 export function decryptKey(options: { privateKey: PrivateKey; passphrase?: string | string[]; config?: PartialConfig }): Promise<PrivateKey>;
 export function encryptKey(options: { privateKey: PrivateKey; passphrase?: string | string[]; config?: PartialConfig }): Promise<PrivateKey>;
 export function reformatKey(options: { privateKey: PrivateKey; userIDs?: UserID|UserID[]; passphrase?: string; keyExpirationTime?: number; config?: PartialConfig }): Promise<KeyPair>;
+export function revokeKey(options: { key: PrivateKey, reasonForRevocation?: { flag?: enums.reasonForRevocation, string?: String }, config?: PartialConfig }): Promise<PrivateKey>;
```

Comment:
@gmreburn we've just introduced a `date` parameter for `revokeKey` in #1302 , could you add it to the definitions as well please? See https://github.com/openpgpjs/openpgpjs/pull/1302/files#diff-f0804beb9a26d785ef733ba8a71c41be04aa299329b68eb114798f3189fe5da4R133